src/bundles/SellerRegistration/redux/helpers.js

```diff
@@ -108,4 +108,19 @@ export const focusHeading = () => {
       const heading = document.querySelector('h1');
       return heading && heading.focus();
   }
-}
\ No newline at end of file
+}
+
+export const isDailyRateMissing = (pricing, services) => {
+  let hasDailyRates = true
+  if (!pricing || !services) {
+    hasDailyRates = false
+  } else {
+    for(let domain of Object.keys(services)) {
+      if (typeof pricing[domain] === 'undefined') {
```

Comment:
I think we may need to check the `maxPrice` property.  I had trouble replicating the message due to empty strings.